oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/backend/MapReducerIgniteLocalPeek.java

```diff
@@ -236,19 +232,12 @@ public Long next() {
       }
     }
 
-    public abstract S execute(Ignite node);
-
     S execute(Ignite node, CellProcessor<S> cellProcessor) {
-      Iterator<Long> cellKeysIterator = new CellKeysIterator(cellIdRanges);
-
       Set<IgniteCache<Long, GridOSHEntity>> caches = this.cacheNames.stream()
           .map(node::<Long, GridOSHEntity>cache)
           .collect(Collectors.toSet());
 
-      return StreamSupport.stream(
-              Spliterators.spliteratorUnknownSize(cellKeysIterator, 0),
-              true
```

Comment:
:thinking: `Streams.stream(…)` in the new version doesn't set the parallel flag. :see_no_evil: